### DISCUSSION TITLE
Is there any api which can let me know if repository belongs to organisation

### DISCUSSION THREAD

**Discussion by desh-naik** [community/community#34419] [Apps, API and Webhooks]
Is there any api which can let me know if repository belongs to organisation

**Comment by LangLangBart** [comment]
`isInOrganization` in [GraphQL API](https://docs.github.com/en/graphql/reference/objects#repository)
<img src="https://user-images.githubusercontent.com/92653266/192724664-2269f018-d7d1-4e20-9346-6264d45978f7.png" width="650">
<img src="https://user-images.githubusercontent.com/92653266/192725201-8dd2236d-5d9e-40a6-b258-deab5029fb67.png" width="600">
